src/Event/Event.php

```diff
@@ -6,15 +6,16 @@
 
 class Event extends BaseEvent
 {
-    /*** @var Session */
+    /** @var Session */
     protected $session;
 
     /**
-     * @param Session
-     * @param string
+     * @param string $name
```

Comment:
Yes, my fault. In my very [first commit](https://github.com/rkrx/sesshin/commit/499bc2bcd4251162b39758ab11a6ea22977c6526), I choose to place $name at the beginning and changed that later without applied the change to the phpdoc.